### PR TITLE
Add IRIS

### LINKED ISSUE #5334: IRIS
**Brand Name:** IRIS
**Website:** https://www.iris.co.uk/
**Alexa rank:** # 381,074 (https://www.alexa.com/siteinfo/iris.co.uk)
  

**Official resources for icon and color:**

Not found yet.

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
iris

**Issue:** Closes #5334
**Similarweb rank:** [144,591](https://www.similarweb.com/website/iris.co.uk/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon from [logo](https://www.iris.co.uk/wp-content/themes/iris/assets/images/footer-iris-logo-white.svg) in [website](https://www.iris.co.uk/) footer. Colour is the background colour of the footer, chosen because it's the primary colour in the [full colour logo](https://www.iris.co.uk/wp-content/themes/iris/assets/images/iris-logo-dark.png).

### CODE REVIEW

**Comment by jorgeamadosoria** [comment]
I think the logo should include the "RIS" part. It doesn't appear without it, and besides without "RIS" this just looks like a signal strength icon in a mobile phone

File: icons/iris.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>IRIS</title><path d="M18.988.135a.22.22 0 00-.22.21l-.01 23.311a.22.22 0 00.22.21h4.815a.207.207 0 00.207-.21V.356a.22.22 0 00-.207-.221zm-9.5 8.293a.22.22 0 00-.22.209L9.25 23.64a.22.22 0 00.22.209h4.805a.207.207 0 00.207-.21V8.638a.22.22 0 00-.207-.21zm-9.3 7.726a.207.207 0 00-.188.207v7.295c0 .115.092.209.207.21h4.805a.22.22 0 00.209-.21v-7.295a.22.22 0 00-.21-.207H.188Z"/></svg>
```
Comment (elliotwutingfeng) [inline icons/iris.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>IRIS</title><path d="M21.08 8.252c-1.491 0-2.55.875-2.55 2.197v.02c0 1.447.948 1.853 2.415 2.228 1.22.313 1.47.522 1.47.928v.02c0 .427-.398.687-1.054.687-.797 0-1.46-.315-2.091-.816l-.043-.036a.063.063 0 0 0-.075.012l-.882 1.049a.068.068 0 0 0 0 .072l.068.059a4.645 4.645 0 0 0 2.986 1.076c1.573 0 2.676-.814 2.676-2.26v-.017c.005-1.273-.828-1.805-2.309-2.19-1.26-.322-1.574-.478-1.574-.957v-.021c0-.354.324-.635.938-.635.577 0 1.174.238 1.779.639l.074.05a.068.068 0 0 0 .074-.02l.774-1.118a.059.059 0 0 0-.008-.069l-.068-.05-.006-.004c-.716-.542-1.585-.844-2.594-.844zm-15.248.103a.068.068 0 0 0-.066.065l-.004 7.16a.068.068 0 0 0 .068.065h1.48a.064.064 0 0 0 .064-.065V8.424a.068.068 0 0 0-.064-.069zm2.918 0a.064.064 0 0 0-.064.065v7.16a.064.064 0 0 0 .064.065h1.475a.064.064 0 0 0 .064-.065v-2.268h1.26l1.558 2.333h1.811c.038 0 .05-.025.031-.053l-1.744-2.551c.926-.344 1.559-1.085 1.559-2.26v-.021c0-.688-.22-1.26-.625-1.666-.478-.477-1.192-.739-2.121-.739zm7.15 0a.064.064 0 0 0-.064.065v7.16a.064.064 0 0 0 .064.065h1.475a.064.064 0 0 0 .064-.065V8.42a.064.064 0 0 0-.064-.065zm-5.597 1.448h1.578c.781 0 1.26.353 1.26 1.04v.022c0 .614-.448 1.031-1.229 1.031h-1.617zm-7.389 1.1a.068.068 0 0 0-.066.064l-.006 4.61a.068.068 0 0 0 .068.062h1.475a.064.064 0 0 0 .064-.063v-4.61a.068.068 0 0 0-.064-.064zm-2.85 2.372A.064.064 0 0 0 0 13.34v2.24a.064.064 0 0 0 .064.065H1.54a.068.068 0 0 0 .065-.065v-2.24a.068.068 0 0 0-.065-.065z"/></svg>
```

**Review by elliotwutingfeng — CHANGES_REQUESTED** [review]
Full logo with "IRIS" text.

**Review by LitoMore — APPROVED** [review]
image

Perfect!

**Pull request merged**